apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/modal.js

```diff
@@ -51,13 +98,20 @@ const GlobalStylesModal = () => {
 		>
 			<div className="wpcom-global-styles-modal__text">
 				<h1 className="wpcom-global-styles-modal__heading">
-					{ __( 'A powerful new way to style your site', 'full-site-editing' ) }
+					{ showUnlockStylesModal
+						? __( 'Publish your new site styles', 'full-site-editing' )
+						: __( 'A powerful new way to style your site', 'full-site-editing' ) }
 				</h1>
 				<p className="wpcom-global-styles-modal__description">
-					{ __(
-						"Change all of your site's fonts, colors and more. Available on any paid plan.",
-						'full-site-editing'
-					) }
+					{ showUnlockStylesModal
+						? __(
+								"To activate your site styles, and unlock advanced design customization tools, you'll need upgrade to a paid plan.",
+								'full-site-editing'
+						  )
+						: __(
+								"Change all of your site's fonts, colors and more. Available on any paid plan.",
+								'full-site-editing'
+						  ) }
```

Comment:
[Nitpick no need to change] Maybe we could set up the translations outside of the actual jsx. That way we wouldn't need to check the value of `showUnlockStyleModal` twice and we could just provide the immutable variables with the translations, we would end up with a much shorter jsx, something like:
```
<div className="wpcom-global-styles-modal__text">
				<h1 className="wpcom-global-styles-modal__heading">{ title }</h1>
				<p className="wpcom-global-styles-modal__description">{ message}</p>
<div className="wpcom-global-styles-modal__actions">
```

Reply:
Good suggestion, done in https://github.com/Automattic/wp-calypso/pull/69477/commits/b0f977c2c3b9e369f72687085bc7aaf4d4ab5992